CONTRIBUTING.md

```diff
@@ -0,0 +1,81 @@
+# How to contribute
+
+WIP
+
+## Clone the repo
+
+```
+> git clone https://github.com/cockroachdb/ui/
+```
+
+The UI repo is a monorepo managed by [Lerna](https://lerna.js.org/). Once you clone the repo, be sure to [bootstrap](). _In the following examples, I'm using `npx`. If you do this very often you may consider installing Lerna globally (`npm i -g lerna`)._
+
+```
+> cd path/to/ui
+> npx lerna bootstrap
+```
+
+## Branch and develop
+
+Create a branch for your work. When you are ready, create a [pull-request](https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/proposing-changes-to-your-work-with-pull-requests) for your work.
+
+```
+> git checkout -b package-name--branch-name-describing-work
+```
+
+Branch names should contain the name of the package you are working on as well as a description of the work you are doing.
```

Comment:
If we are using forks then the name of the branch doesn't really matter.

Reply:
Nice catch. I will change this section to propose _fork and pull_ over _branch and pull_.